ingest/tutorial/example_common.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/stellar/go/ingest/ledgerbackend"
+)
+
+var (
+	config = ledgerbackend.CaptiveCoreConfig{
+		// Change these based on your environment:
+		BinaryPath:        "/usr/local/bin/stellar-core",
+		ConfigAppendPath:  "stellar-core-stub.toml",
+		NetworkPassphrase: "Test SDF Network ; September 2015",
+		HistoryArchiveURLs: []string{
+			"https://history.stellar.org/prd/core-testnet/core_testnet_001",
```

Comment:
While we support only one archive now I think it's worth using a config that will be used in the future.
```suggestion
			"https://history.stellar.org/prd/core-testnet/core_testnet_001",
			"https://history.stellar.org/prd/core-testnet/core_testnet_002",
			"https://history.stellar.org/prd/core-testnet/core_testnet_003",
```